scripts/apim/deploy.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```

Comment:
Shall we change the license header to the open source licensing?

Reply:
Added the Apache licence to shell scripts